_dev/coding-style.md

```diff
@@ -6,8 +6,65 @@ description: The preferred style for code committed to NeoMutt
 
 # Coding Style
 
-Every project has its own coding style, this one is no exception.
-However, I haven't written down the rules.  For now, ...
+## at the moment (pre-fork)
 
-- Trailing whitespace is **evil**
+Generally speaking, we don't want to diverge much from upstream mutt. We're
+still merging patches in - and if we didn't follow upstream's coding style,
+merging would be much harder.
```

Comment:
@guyzmo I'm not sure if you've understood me - i am talking about the mutt -> neomutt direction here, not the other way around.

> As long as we're not modifying the existing code, it does not matter much what style we apply for new code.

True, but having two different style for things which still receive patches from mutt and one fro new code from us would be not very beautiful to look at. :)